qemu/tests/fio_windows.py

```diff
@@ -0,0 +1,105 @@
+            except (aexpect.ShellStatusError, aexpect.ShellTimeoutError):
+                pass
+
+        count = 0
+        while count < 3:
+            ret = session_cmd(session, cmd)
+            if ret is not None:
+                return ret
+            count += 1
+        return None
+
+    tmp_dir = data_dir.get_tmp_dir()
+    install_path = params["install_path"].rstrip("\\")
+    start_cmd = params.get("start_cmd")
+    stop_cmd = "taskkill /T /F /IM fio.exe"
+    stop_cmd = params.get("stop_cmd", stop_cmd)
+    check_running_cmd = "tasklist|findstr /I fio"
+    check_running_cmd = params.get("check_running_cmd", check_running_cmd)
+    test_installed_cmd = 'dir "%s"|findstr /I fio' % install_path
+    test_installed_cmd = params.get("check_installed_cmd", test_installed_cmd)
+
+    vm = env.get_vm(params["main_vm"])
+    vm.verify_alive()
+    timeout = float(params.get("login_timeout", 360))
+    session = vm.wait_for_login(timeout=timeout)
+    session_serial = vm.wait_for_serial_login(timeout=timeout)
+    try:
+        installed = session.cmd_status(test_installed_cmd) == 0
+        if not installed:
+            download_url = params.get("download_url")
+            if download_url:
+                dst = r"c:\\"
+                pkg_md5sum = params["pkg_md5sum"]
+                error.context("Download fio.msi file", logging.info)
+                pkg = utils.unmap_url_cache(tmp_dir,
+                                            download_url, pkg_md5sum)
+                vm.copy_files_to(pkg, dst)
+            else:
+                dst = r"%s:\\" % utils_misc.get_winutils_vol(session)
+
+            error.context("Install fio in guest", logging.info)
+            install_cmd = params["install_cmd"]
+            install_cmd = re.sub(r"DRIVE:\\+", dst, install_cmd)
+            session.cmd(install_cmd)
+            time.sleep(30)
+            config_cmd = params.get("config_cmd")
+            if config_cmd:
+                session.cmd(config_cmd)
+
+        error.context("Start fio in guest", logging.info)
+        start_cmd = params["start_cmd"]
+        test_timeout = int(params.get("timeout", "360"))
+        logging.info("fio cmd: %s" % start_cmd)
+        session.sendline(start_cmd)
+        time.sleep(10)
+        if not loop_session_cmd(session_serial, check_running_cmd):
+            raise error.TestError("fio process is not started")
+
+        error.context("Verify vm is alive", logging.info)
+        utils_misc.wait_for(vm.verify_alive,
+                            timeout=test_timeout, step=30)
+    finally:
+        error.context("Stop load and clean tmp files", logging.info)
+        if not installed and download_url:
+            utils.system("rm -f %s/fio*.msi" % tmp_dir)
+            session.cmd("del /f /s %sfio*.msi" % dst)
+        if loop_session_cmd(session_serial, check_running_cmd):
+            if not loop_session_cmd(session_serial, stop_cmd):
```

Comment:
I mean:
kill process can be successful, but the previous execute fio cmd would be failed.
Will it raise some error?